packages/sheets-formula/src/locale/function-list/text/en-US.ts

```diff
@@ -179,8 +179,9 @@ export default {
             },
         ],
         functionParameter: {
-            number1: { name: 'number1', detail: 'first' },
-            number2: { name: 'number2', detail: 'second' },
+            text1: { name: 'findText', detail: 'The text to find. The item can be a text value, number, or cell reference.' },
```

Comment:
key as `findText`, name as `find_text`

reference https://univer.ai/guides/sheet/customization/formula#how-to-implement-the-formula
![image](https://github.com/dream-num/univer/assets/26371161/d36c173b-2df3-448d-a20f-122638269ade)

like SUMIF:  https://github.com/dream-num/univer/blob/adc1e1f8e572cb29d26d113352f162cbc5984ba4/packages/sheets-formula/src/locale/function-list/math/en-US.ts#L1050